### PR TITLE
Handle null hapiVersion in blocks API

### LINKED ISSUE #6069: hapi_version is null.null.null on blocks API
### Description

Blocks API for older record files that did not have `hapi_version` populated in the protobuf show `"hapi_version":"null.null.null"` instead of `"hapi_version": null`.

### Steps to reproduce

https://mainnet-public.mirrornode.hedera.com/api/v1/blocks?limit=1&order=asc

### Additional context

_No response_

### Hedera network

mainnet

### Version

0.80.0

### Operating system

None

### PR DESCRIPTION
**Pull request by MarcKriguerAtHedera**
**Description**:

* Fix `recordFile.getFullVersion` to return `null` instead of `null.null.null`
* New spec file to test that it works

**Related issue(s)**:

Fixes #6069 

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit)

### CODE REVIEW

File: hedera-mirror-rest/model/recordFile.js [hunk 1]
```diff
@@ -62,7 +62,8 @@ class RecordFile {
   }
 
   getFullHapiVersion() {
-    return `${this.hapiVersionMajor}.${this.hapiVersionMinor}.${this.hapiVersionPatch}`;
+    return `${this.hapiVersionMajor}` == `null` ? null
```
Comment (steven-sheehy) [inline hedera-mirror-rest/model/recordFile.js:65]:
```suggestion
    return hapiVersionMajor === null || hapiVersionMinor === null || hapiVersionPatch === null ? null
```

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-rest/model/recordFile.js (hunk 1 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-rest/model/recordFile.js]: Done/

Comment (MarcKriguerAtHedera) [reply hedera-mirror-rest/model/recordFile.js]: Redone (the backticks and $ and braces are still gone, but needed to restore the "`this`." for the code to work).

File: hedera-mirror-rest/model/recordFile.js [hunk 2]
```diff
@@ -62,7 +62,8 @@ class RecordFile {
   }
 
   getFullHapiVersion() {
-    return `${this.hapiVersionMajor}.${this.hapiVersionMinor}.${this.hapiVersionPatch}`;
+    return (this.hapiVersionMajor === null || this.hapiVersionMinor === null || this.hapiVersionPatch === null) ? null
```
Comment (xin-hedera) [inline hedera-mirror-rest/model/recordFile.js:65]: add `nullable: true` to open api spec for `hapi_version`

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: hedera-mirror-rest/model/recordFile.js (hunk 2 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-rest/model/recordFile.js:65]: Done.

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**